ipet/parsing/ReaderManager.py

```diff
@@ -226,12 +227,21 @@ def updateProblemName(self, line, currentcontext, readers):
         """
         sets up data structures for a new problem if necessary
         """
+        self.updateLineNumberData(line[0], currentcontext, "LineNumbers_Begin")
+
         problemname, problempath = self.getProblemName(line[1])
+        if line[1].split()[2] == "==MISSING==":
```

Comment:
This "==MISSING==" adds a new logfile layout restriction that needs to be maintained. I would personally prefer to leave this somehow out. Let's discuss this again...

Reply:
I made a proposition